docs/blog2/src/guide.md

```diff
@@ -77,11 +77,11 @@ The flowchart above may help you understand how the plugin works and it's design
 
 ## Collecting Articles and Generating Info
 
-To get started, the plugins will pick those pages which neeeds to be tricted as articles. This is only the first step to drop those pages you don’t want.
+To get started, the plugins will pick those pages which need to be treated as articles. This is only the first step to drop those pages you don’t want.
 
 ::: note
 
-By default, all the pages generated from Markdown files but not homepage will be included as articles.
+By default, all the pages generated from Markdown filesa apart from homepage be included as articles.
```

Comment:
What's `filesa`? Is it a exist english word?

Reply:
sorry didn't see that.Let me fix that